doc/src/DebugAndLogging.md

```diff
@@ -137,6 +137,7 @@ The `ROCBLAS_LAYER` environmental variable can be set to output GEMM information
 Additionally, using environment variable "MIOPEN_GEMM_ENFORCE_BACKEND", can override the default behavior. The default behavior which is to use
 both MIOpenGEMM and rocBlas depending on the input configuration:
 
+* `MIOPEN_GEMM_ENFORCE_BACKEND=0`, use MIOpenTensile for FP32, use rocBLAS for FP16 if enabled
```

Comment:
Please use value 4. When env var is unset, `Value(ENV_VAR)` returns 0.

Reply:
Will do